src/utils/venue.ts

```diff
@@ -63,6 +64,7 @@ export const createJazzbar = (values: FormValues): JazzbarVenue => {
   return {
     template: VenueTemplate.jazzbar,
     name: values.name || "Your Jazz Bar",
+    slug: values.name ? createSlug(values.name) : createSlug("Your Jazz Bar"),
```

Comment:
the `createSlug("Your Jazz Bar")` can probably be calculated ones, before component is created, used here as `DEFAULT_JAZZ_BAR_SLUG`, but then, what about the following default? Maybe a function like `generateId()` that will just add numbers `-1`, `-2`...

Reply:
These fields are not used anywhere. The function is legacy and it was created for another purpose. Right now it's used in 2 places and only 2 fields from this whole object are being used, and they are both in `config.landingPage...`

Reply:
I'm fixing this only because the slug is now a required field in the Venue type and TS is crying.